src/App.jsx

```diff
@@ -14,33 +19,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function sortGoods(goods, sortField, reversed) {
+  const sortedGoods = [...goods].sort((good1, good2) => {
+    switch (sortField) {
+      case SORT_FIELD_ALPHABET:
+        return good1.localeCompare(good2);
+      case SORT_FIELD_LENGTH:
+        return good1.length - good2.length;
+      default:
+        return 0;
+    }
+  });
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  return reversed ? sortedGoods.reverse() : sortedGoods;
+}
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The initial state of `sortField` is set to an empty string. This can cause the `sortGoods` function to return unsorted goods since the empty string is not a valid sort field. Consider initializing `sortField` with a default value like `SORT_FIELD_ALPHABET` or `SORT_FIELD_LENGTH` to ensure the goods are sorted initially.